src/connections/google/auth.js

```diff
@@ -0,0 +1,16 @@
+const { JWT } = require('googleapis').google.auth;
+const { client_email, private_key } = require('../../env').GOOGLE_APPLICATION_CREDENTIALS; // eslint-disable-line camelcase
+
+/**
+ *
+ * @param {Array} endpoints Google API endpoints to be authorized for this client
+ * @returns JWT
+ */
+module.exports = async (endpoints = []) => {
+  const client = new JWT(client_email, null, private_key, endpoints);
+  await client.authorize((err) => {
```

Comment:
I don't think this will actually await, as the client.authorize call will immediately execute, and the Promise.resolve function will not be immediately returned. Instead, you should wrap the authorize call with a promise beforehand, and then await:

```js
const authorize = new Promise((resolve, reject) => {
  client.authorize((err) => {
    if (err) {
      reject(err);
    } else {
      resolve();
    }
  })
}); 
await authorize();
```